web/scripts/build-web.mjs

```
@@ -162,22 +172,33 @@ function composeVersionID() {
 function createEntryPointOptions([source, dest], overrides = {}) {
     const outdir = path.join(__dirname, "..", "dist", dest);
 
-    return {
-        ...BASE_ESBUILD_OPTIONS,
+    /**
+     * @type {esbuild.BuildOptions}
+     */
+
+    const entryPointConfig = {
         entryPoints: [`./src/${source}`],
         entryNames: `[dir]/[name]-${composeVersionID()}`,
+        publicPath: path.join("/static", "dist", dest),
         outdir,
-        ...overrides,
     };
+
+    /**
+     * @type {esbuild.BuildOptions}
+     */
+    const mergedConfig = deepmerge(BASE_ESBUILD_OPTIONS, entryPointConfig, overrides);
+
+    return mergedConfig;
 }
 
 /**
  * Build all entry points in parallel.
  *
  * @param {EntryPoint[]} entryPoints
+ * @returns {Promise<esbuild.BuildResult[]>}
  */
 async function buildParallel(entryPoints) {
-    await Promise.allSettled(
+    return Promise.all(
```

Fixes issue where build errors are swallowed during compile.